components/ruby/bundler/Makefile

```diff
@@ -0,0 +1,58 @@
+
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright 2016 Grüninger GmbH, Andreas Grüninger (grueni)
+#
+include ../../../make-rules/shared-macros.mk
+
+RUBY_VERSION = 2.2
+RUBY_LIB_VERSION = 2.2.0
```

Comment:
I think that 
```
diff --git a/make-rules/shared-macros.mk b/make-rules/shared-macros.mk
index 8df28d3..d34fe8e 100644
--- a/make-rules/shared-macros.mk
+++ b/make-rules/shared-macros.mk
@@ -567,9 +567,14 @@ F77 =              $(F77.$(COMPILER).$(BITS))
 FC =           $(FC.$(COMPILER).$(BITS))
 
 RUBY_VERSION =  1.9
-RUBY_LIB_VERSION =      1.9.1
+RUBY_LIB_VERSION.1.9 = 1.9.1
+RUBY_LIB_VERSION.2.2 = 2.2.0   
+RUBY_LIB_VERSION.2.3 = 2.3.0
 RUBY.1.9 =      /usr/ruby/1.9/bin/ruby
+RUBY.2.2 =     /usr/ruby/2.2/bin/ruby
+RUBY.2.3 =     /usr/ruby/2.3/bin/ruby
 RUBY =          $(RUBY.$(RUBY_VERSION))
+RUBY_LIB_VERSION = $(RUBY_LIB_VERSION.$(RUBY_VERSION))
``` 
is not so a big change to avoid doing it in this PR, especially that component can leverage it.